src/shared/components/ScatterPlot.tsx

```diff
@@ -74,14 +76,22 @@ const ScatterPlot: FunctionComponent<Props> = ({
     legendOrientationThreshold
   )
 
+  let timerange = null
+
+  if (xColumn === '_time') {
+    timerange = timeRange
+  }
+
   const [xDomain, onSetXDomain, onResetXDomain] = useVisXDomainSettings(
     storedXDomain,
-    table.getColumn(xColumn, 'number')
+    table.getColumn(xColumn, 'number'),
+    ...timerange
```

Comment:
I think this will throw an error when `timerange` is null because you cannot spread a null value into the function call arguments.

Reply:
alright i went ahead and fixed this, so now it defaults not to null